pkg/operator/configobservation/apiserver/observe_termination_duration.go

```diff
@@ -82,8 +86,14 @@ func ObserveWatchTerminationDuration(genericListers configobserver.Listers, _ ev
 		// we got an error so without the infrastructure object we are not able to determine the type of platform we are running on
 		return existingConfig, append(errs, err)
 	}
-	switch infra.Spec.PlatformSpec.Type {
-	case configv1.AWSPlatformType:
+
+
+	switch {
+	case infra.Status.ControlPlaneTopology == configv1.SingleReplicaTopologyMode:
+		// reduce termination duration from 135s (default) to 60s to reach the maximum downtime for SNO
+		// on a single node the shutdown-delay-duration is set to 0s, additional 60s is for in-flight requests
+		observedGracefulTerminationDuration = "60"
```

Comment:
I would actually set it to something slightly above `60s` or `135` in general

Reply:
we have a Jira for 10s.